commons/tests/validators_test.py

```diff
@@ -0,0 +1,17 @@
+# pylint: disable=no-self-use
+
+import pytest
+from c2cgeoportal_commons.lib.validators import url
+
+
+class TestUrl():
+
+    @pytest.mark.parametrize("valid_url", [
+        'http://geomapfish.org',
+        'https://geomapfish.org/functionalities',
+        'geomapfish.org/functionalities',
+        'static://img/cadastre.jpeg',
```

Comment:
@arnaud-morvan Shouldn't you also test `static:///...` as described here https://github.com/camptocamp/c2cgeoportal/blob/2.3/doc/integrator/customise.rst#url

Reply:
Yes it would make sense. Note that the regex accept anything except whitespaces after "static://".